news/_posts/2020-08-12-write-product-specifications.md

```diff
@@ -0,0 +1,79 @@
+
+
+### Add missing specifications
+
+Many existing features have not been formally specified yet so there is still a lot to do. Adding missing specifications about the current behaviors will help to provide the PrestaShop ecosystem with a precise view of the software’s functional scope.
+
+
+### Specify new features
+
+When a new feature is suggested or planned, it is essential to define its scope and therefore specify its use since it will guide its developments.
+
+
+### How-to
+
+Now let us focus on the practical side! Starting to write a specification, you have two choices: the page already exists in the specifications repository and you just have to edit it to complete it, or it does not exist yet and then you will have to create it to suggest your specifications.
+
+Whether it is an edition or addition, a pull request (PR) will have to be created - just like for any development! You will learn more about the detailed steps in the `CONTRIBUTING.md` file [here](https://github.com/PrestaShop/prestashop-specs/blob/master/CONTRIBUTING.md#process-to-follow) and the suggested template to use in the next section of this post.
+
+![PrestaShop Product Specifications Management](/assets/images/2020/08/prestashop-specs-management.jpg)
+
+Once your specifications have been submitted, the pull request is reviewed by the core team and validated by the product team. If you specify a new feature, specifications will have to be validated by the core team (product, dev, and QA if need be) and merged before starting the development. If any behavior is redefined during the development phase, the specifications must be updated accordingly following the same workflow: PR > validation > merge > development.
+
+
+## Anatomy of the product specifications
+
+Writing product specifications requires using Git and GitHub, it works pretty much the same as a code contribution. You can read the detailed steps in the `CONTRIBUTING.md` file right [here](https://github.com/PrestaShop/prestashop-specs/blob/master/CONTRIBUTING.md). You can refer to the template below to structure your contribution, and obviously you are greatly encouraged to have a look at the already written product specifications. ;-)
+
+![PrestaShop Product Specifications Template](/assets/images/2020/08/prestashop-specs-template.jpg)
+
+And that’s it! Product specifications are a nice way to contribute to PrestaShop and writing it is accessible to everyone: developers who need the functional scope of a feature to start working on it, merchants who use the software every day and have a deep understanding of how pages work, all kinds of contributors who want to get familiar with the PrestaShop open source project and reinforce the knowledge (and the business) around the software. 
```

Comment:
This makes no sense, only people who are very familiar with the software should contribute with specifications.

Reply:
I change it, let me know if it's ok for you